src/smc-hub/postgres-base.coffee

```diff
@@ -18,6 +18,10 @@ DEFAULT_TIMEOUT_DELAY_MS = DEFAULT_TIMEOUS_MS * 4
 
 QUERY_ALERT_THRESH_MS=5000
 
+# this is a limit for each query, unless timeout_s is specified.
+# https://postgresqlco.nf/en/doc/param/statement_timeout/
+DEFAULT_STATEMENT_TIMEOUT_s = 30
```

Comment:
Rename this from `DEFAULT_STATEMENT_TIMEOUT_s` to  `DEFAULT_STATEMENT_TIMEOUT_S`.  The final `s` should be uppercase.  E.g., look just two lines above at `QUERY_ALERT_THRESH_MS` where the `MS` is uppercase, not `ms`.

Reply:
This is especially important since this is evil coffeescript code, where typos just break stuff at runtime...